src/Android/Services/AndroidPushNotificationService.cs

```diff
@@ -52,6 +55,31 @@ public Task UnregisterAsync()
             // Do we ever need to unregister?
             return Task.FromResult(0);
         }
+
+        public void DismissLocalNotification(string notificationId)
+        {
+            var notificationManager = NotificationManagerCompat.From(Android.App.Application.Context);
+            notificationManager.Cancel(int.Parse(notificationId));
+        }
+
+        public void SendLocalNotification(string title, string message, string notificationId = null)
+        {
+            notificationId = notificationId ?? new Random().Next(1000, 999999).ToString();
+            var context = Android.App.Application.Context;
+            var intent = new Intent(context, typeof(MainActivity));
+            var pendingIntent = PendingIntent.GetActivity(context, new Random().Next(1000, 999999), intent, PendingIntentFlags.OneShot);
```

Comment:
I believe you can use a const `requestCode` for `GetActivity` or did you have any issues with being it constant?

Reply:
Yes the intent is always the same and we actually only the keep the most up to date, so we can use the `requestCode` as a constant.